org.flatpak.gnomeConnectionManager.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.flatpak.gnomeConnectionManager
+runtime: org.gnome.Platform
+  #runtime-version: '3.38'
+runtime-version: '43'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --filesystem=home
+sdk: org.gnome.Sdk
+command: /app/bin/gnome_connection_manager.py
+modules:
+  - name: vte
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/vte/0.72/vte-0.72.1.tar.xz
+        sha256: 0554f9f88d56ce2d78398fcc7f69bc00e53bbbc5f694e0ae1dcaf5286f89d7e4
+  - name: gtk
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtk+/3.24/gtk%2B-3.24.37.tar.xz
+        sha256: 6745f0b4c053794151fd0f0e2474b077cccff5f83e9dd1bf3d39fe9fe5fb7f57
+  - name: libcanberra
```

Comment:
please use the shared modules. https://github.com/flathub/shared-modules/ as libcanberra need some patches.